### DISCUSSION TITLE
error notification in prefect 2.0

### DISCUSSION THREAD

**Discussion by ghislainp** [PrefectHQ/prefect#5932] [Q&A]
It's probably a stupid question, but I don't find a way to notify errors in task/flow. The UI log shows the error and the status of the flow has this information, but I'd like to connect a function/task to send notification with detailed information on the error. Prefect 1.0 has state_handlers (for task, but for flow??) and I've found nothing in prefect 2.0. Is it because the implementation of state_handlers is to come, or because there is another elegant way to report errors ?

**Comment by zanieb** [comment]
Hi! We're working on more elegant ways to report errors :) you'll see some first steps for this in the near future.
